plugins/generator-1.15.2/forge-1.15.2/procedures/entity_was_shot.java.ftl

```diff
@@ -0,0 +1 @@
+((projectileentity != null)?(projectileentity instanceof ${generator.map(field$projectiles, "projectiles")}):false)
```

Comment:
> != null and instanceof serve the same purpose
null is not the instance of any class
(projectileentity instanceof ${generator.map(field$projectiles, "projectiles")}) is enough

Reply:
this can be replaced with `projectileentity instanceof ${generator.map(field$projectiles, "projectiles")}`

Reply:
`!= null` part has been removed